internal/render/nodeconfigurator/pod.go

```diff
@@ -0,0 +1,80 @@
+package nodeconfigurator
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	slurmv1alpha1 "nebius.ai/slurm-operator/api/v1alpha1"
+	"nebius.ai/slurm-operator/internal/consts"
+	"nebius.ai/slurm-operator/internal/render/common"
+)
+
+func renderPodSpec(nodeConfigurator slurmv1alpha1.NodeConfiguratorSpec) corev1.PodSpec {
+	hostPID := nodeConfigurator.Rebooter.Enabled
```

Comment:
maybe we can add here some comment, that we need host pids here, because we want to reboot host with it. Or what is the reason?